### PR TITLE
Allow events to be shown as Busy or Free.

### PR DESCRIPTION
**Pull request by laura-digio**
Allow events to be shown as Busy or Free.

### CODE REVIEW

**Comment by laura-digio** [comment]
@Brett09 @nickrandolph Could you review this PR?

**Comment by laura-digio** [comment]
@MaikuB Would you review this change?

**Comment by MaikuB** [comment]
@laura-digio sorry I'm not part of the organisation since I don't work at Built to Roam for a while. I believe it's up to @nickrandolph now

**Comment by laura-digio** [comment]
Hi there! @nickrandolph could you review this pull request whenever possible? Thanks a lot!

**Comment by nickrandolph** [comment]
@laura-digio apologies for the delay - going through and reviewing now. Would you be able to clean up the PR to remove changes that aren't related to the PR (eg changes to project.pbxproj). I'll endeavor to get this reviewed asap.

**Comment by laura-digio** [comment]
@ni

> @laura-digio apologies for the delay - going through and reviewing now. Would you be able to clean up the PR to remove changes that aren't related to the PR (eg changes to project.pbxproj). I'll endeavor to get this reviewed asap.

I've restored the affected files to its previous state.

File: android/src/main/kotlin/com/builttoroam/devicecalendar/common/Constants.kt [hunk 1]
```diff
@@ -62,7 +63,9 @@ class Constants {
                 CalendarContract.Events.EVENT_LOCATION,
                 CalendarContract.Events.CUSTOM_APP_URI,
                 CalendarContract.Events.EVENT_TIMEZONE,
-                CalendarContract.Events.EVENT_END_TIMEZONE
+                CalendarContract.Events.EVENT_END_TIMEZONE,
+                CalendarContract.Events.AVAILABILITY,
+                CalendarContract.Events.STATUS
```
Comment (nickrandolph) [inline android/src/main/kotlin/com/builttoroam/devicecalendar/common/Constants.kt:68]: @laura-digio I noticed that you added .STATUS here - is this needed in order to get availability information? or was there another bug that this is fixing?

Comment (alvaroDigio) [reply android/src/main/kotlin/com/builttoroam/devicecalendar/common/Constants.kt]: @nickrandolph sorry, it's not necessary. I fix this change!
Sorry and thanks for look the PR

**Comment by nickrandolph** [comment]
Hi @laura-digio & @alvaroDigio I've just been going through this PR - thanks again for the updates. I was just wondering whether we should be making the Availability property in dart an enum instead of a string. I'd suggest the enum values could be free, busy, tentative, unavailable (null for any other state).

**Comment by nickrandolph** [comment]
Thanks @laura-digio and @alvaroDigio for the contribution - if you can help out with #235 that would be awesome

**Comment by alvaroDigio** [comment]
@nickrandolph thanks for your hard work. When we can, we will make the change #235 
Thanks!

**Pull request merged**